### ISSUE TITLE
DAgger training problem

### ISSUE THREAD

**Issue by ccccggggf** [mihdalal/manipgen#16]
I encounter the 'core dump' probem when running your 'Train DAgger policy' code, which I show below in detail. Even I reduce the batchsize and buffer size, the problem is the same.
        Importing module 'gym_38' (/data1/lipeida/isaacgym/python/isaacgym/_bindings/linux-x86_64/gym_38.so)
        Setting GYM_USD_PLUG_INFO_PATH to /data1/lipeida/isaacgym/python/isaacgym/_bindings/linux-x86_64/usd/plugInfo.json
        PyTorch version 2.1.1+cu121
        Device count 8
        /data1/lipeida/isaacgym/python/isaacgym/_bindings/src/gymtorch
        Using /home/lipeida/.cache/torch_extensions/py38_cu121 as PyTorch extensions root...
        Emitting ninja build file /home/lipeida/.cache/torch_extensions/py38_cu121/gymtorch/build.ninja...
        Building extension module gymtorch...
        Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
        ninja: no work to do.
        Loading extension module gymtorch...
        2025-05-13 11:18:39,914 - INFO - logger - logger initialized
        <unknown>:3: DeprecationWarning: invalid escape sequence \*
        Error: FBX library failed to load - importing FBX data will not succeed. Message: No module named 'fbx'
        FBX tools must be installed from https://help.autodesk.com/view/FBX/2020/ENU/?guid=FBX_Developer_Help_scripting_with_python_fbx_installing_python_fbx_html
        /data1/lipeida/isaacgym/python/isaacgym/torch_utils.py:135: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
        Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
          def get_axis_params(value, axis_idx, x_value=0., dtype=np.float, n_dims=3):
        manipgen_robomimic WARNING(
            No private macro file found!
            It is recommended to use a private macro file
            To setup, run: python /data1/lipeida/manipgen/manipgen_robomimic/manipgen_robomimic/scripts/setup_macros.py
        )
        /home/lipeida/miniconda3/envs/lpd/lib/python3.8/site-packages/hydra/_internal/hydra.py:119: UserWarning: Future Hydra versions will no longer change working directory at job runtime by default.
        See https://hydra.cc/docs/1.2/upgrades/1.1_to_1.2/changes_to_job_working_dir/ for more information.
          ret = run_job(
        Setting seed: 3407
        
        ============= Initialized Observation Utils with Obs Spec =============
        
        using obs modality: low_dim with keys: ['state']
        using obs modality: rgb with keys: []
        using obs modality: depth with keys: ['visual']
        using obs modality: scan with keys: []
        /home/lipeida/miniconda3/envs/lpd/lib/python3.8/site-packages/gym/spaces/box.py:84: UserWarning: WARN: Box bound precision lowered by casting to float32
          logger.warn(f"Box bound precision lowered by casting to {self.dtype}")
        [Warning] [carb.gym.plugin] useGpu is set, forcing single scene (0 subscenes)
        Not connected to PVD
        +++ Using GPU PhysX
        Physics Engine: PhysX
        Physics Device: cuda:7
        GPU Pipeline: enabled
        段错误 (核心已转储)
